Source/JavaScriptCore/runtime/JSGlobalObject.cpp

```diff
@@ -2596,6 +2599,58 @@ void JSGlobalObject::installTypedArrayConstructorSpeciesWatchpoint(JSTypedArrayV
     m_typedArrayConstructorSpeciesWatchpoint->install(vm);
 }
 
+void JSGlobalObject::installTypedArrayIteratorProtocolWatchpoint(JSObject* base, TypedArrayType typedArrayType)
+{
+    VM& vm = this->vm();
+
+    DeferTermination deferScope(vm);
```

Comment:
nit: let's use DeferTerminationForAWhile instead.   It's probably a good idea to always use DeferTerminationForAWhile instead of DeferTermination (which can throw the TerminationException on destruction).  Maybe we should get rid of DeferTermination as I can't think of any reason why we would really need it.

Reply:
Right, changed.